src/main/kotlin/com/vanniktech/maven/publish/MavenPublishTarget.kt

```diff
@@ -31,4 +34,13 @@ data class MavenPublishTarget(
    * @since 0.7.0
    */
   var signing: Boolean = true
-)
+) {
+
+  val taskName get(): String {
```

Comment:
Are there tests for this given that the API is public?

Reply:
I'll add them. I wanted to make it internal, but Groovy didn't like that for some reason.